constants.py

```diff
@@ -446,3 +454,50 @@
 kCameraServoPGain = 0.15
 kCameraServoIGain = 0.0
 kCameraServoDGain = 0.0
+
+# shooter parameters
+kTurretMotorId = 22
+kShootingMotorId = 23
+kHoodMotorId = 21
+
+kSimTurretMotorPort = 9
+kSimShootingMotorPort = 18
+kSimHoodMotorPort = 19
+
+kSimTurretMinimumLimitSwitchPort = 0
```

Comment:
Just to clarify, the real life limit switches don't need any code, correct?

Reply:
no they rely simply on the falcons attached to them and the limit switch wrapper takes care of it